docroot/profiles/favrskov/modules/custom/favrskov_helper/feeds_tamper_plugins/date_converter.inc

```diff
@@ -36,8 +36,13 @@ function favrskov_helper_plugin_date_to_proper_format_form($importer, $element_k
  * Callback that will be called during import on field that plugin was attached.
  */
 function favrskov_helper_plugin_date_to_proper_format_callback($result, $item_key, $element_key, &$field, $settings) {
-  $date       = new DateTime($field);
-  $timestamp  = $date->getTimestamp();
 
-  $field = format_date($timestamp, $settings['type']);
+  $current_local = locale_get_default();
+  setlocale(LC_ALL,"da_DK");
+  $date = DateTime::createFromFormat("d. M Y", $field);
```

Comment:
@mnakov Are you sure about the dot as you mentioned that some days may not have it ?

Reply:
@vzlatev, I'm sure about the dot, the date always comes with the dot. You can proceed with merge, if everything else seems good to you.